koku/masu/processor/aws/aws_report_parquet_processor.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, manifest_id, account, s3_path, provider_uuid, parquet_local_p
             "lineitem_unblendedrate",
             "lineitem_blendedcost",
             "lineitem_blendedrate",
-            "savingsplan_savingsplaneffectivecost",
+            "calculated_amortized_cost",
```

Comment:
We want to keep the `savingsplan_savingsplaneffectivecost` as the calculated value is just being sent to Postgres, and not in Trino.